### PR TITLE
[bluetooth.ruuvitag] Corrected movement counter channel description

### PR DESCRIPTION
**Pull request by ssalonen**
Signed-off-by: Sami Salonen <ssalonen@gmail.com>

### CODE REVIEW

File: bundles/org.openhab.binding.bluetooth.ruuvitag/src/main/resources/OH-INF/thing/ruuvitag.xml [hunk 1]
```diff
@@ -75,7 +75,7 @@
 	</channel-type>
 	<channel-type id="ruuvitag_movementCounter" advanced="true">
 		<item-type>Number:Dimensionless</item-type>
-		<label>Measurement Counter</label>
+		<label>Movement Counter</label>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.bluetooth.ruuvitag/src/main/resources/OH-INF/thing/ruuvitag.xml:78]: Please also fix the entry in the default translations properties.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.bluetooth.ruuvitag/src/main/resources/OH-INF/thing/ruuvitag.xml (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.bluetooth.ruuvitag/src/main/resources/OH-INF/thing/ruuvitag.xml:78]: https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.bluetooth.ruuvitag/src/main/resources/OH-INF/i18n/bluetooth.properties#L20

Comment (ssalonen) [reply bundles/org.openhab.binding.bluetooth.ruuvitag/src/main/resources/OH-INF/thing/ruuvitag.xml:78]: https://github.com/ssalonen/openhab2-addons/commit/93c9c6dd1962e77a78c9b9d4f141b18c34902208

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**